select_command/_index.md

```diff
@@ -0,0 +1,60 @@
+
+# <center><b>Mission 1: Save the Planet of Fun From Boredom!</b></center>

```

Comment:
Please check out the other files in this repo - there's some special lines that need to appear at the top of the page.

Something like (change Java -> SQL):
---
title: "Java: Basics"
description: "An introduction to the basics of Java programming language."
date: 2020-07-07T00:00:00Z
prereq: "none"
difficulty: "Beginner"
icon: "fab fa-java"
draft: false
---

Reply:
I'm not sure what to put for the icon, I'll leave it blank for now.